caf-audit/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java

```diff
@@ -328,38 +328,94 @@ public void testMultiTenantESIndexing() throws Exception
             //  Send audit event message to Elasticsearch.
             auditEventBuilder.send();
 
-            // TODO: Search across all indexes for appId. The index names of each hit returned should match each tenantId.
-            //  Search across all indices for the applicationId field in Elasticsearch and verify that each hit returned belongs to the indices we expect.
+            // Search across all indices for the applicationId field in Elasticsearch and verify that the expected
+            // number of hits are returned.
             try (TransportClient transportClient
                          = ElasticAuditTransportClientFactory.getTransportClient(esHostAndPort, ES_CLUSTERNAME)) {
 
-                SearchHit[] tenant1IndexHits = new SearchHit[0];
+                String[] tenantIndexIds = new String[2];
+                tenantIndexIds[0] = ES_INDEX_PREFIX + tenant1Id;
+                tenantIndexIds[1] = ES_INDEX_PREFIX + tenant2Id;
 
-                String tenant1IndexId = ES_INDEX_PREFIX + tenant1Id;
-
-                tenant1IndexHits = searchDocumentInIndex(transportClient, tenant1IndexId,
+                SearchHit[] tenantIndicesHits = searchDocumentInIndices(transportClient, tenantIndexIds,
                         APP_ID_FIELD.concat(KEYWORD_SUFFIX), testApplicationId);
 
-                //  Expecting a single hit.
-                Assert.assertTrue(tenant1IndexHits.length == 1);
+                //  Expecting a two hits.
+                Assert.assertTrue(tenantIndicesHits.length == 2);
 
-                SearchHit[] tenant2IndexHits = new SearchHit[0];
+                //  Delete test indexes after verification is complete.
+                deleteTenantIndices(transportClient, tenantIndexIds);
+            }
+        }
+    }
 
-                String tenant2IndexId = ES_INDEX_PREFIX + tenant2Id;
+    private void deleteTenantIndices(TransportClient client, String[] indices) {
```

Comment:
IMO I would rename this method to deleteIndex, deleteIndexes or deleteIndices. Drop the tenant bit.